--- Running tornado.tcpserver.TCPServer#9NN/TCPServerNN, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'tornado.tcpserver.TCPServer#9NN'), ('notebook', 'TCPServerNN'), ('text', '  File "TCPServerNN.py", line 3, in <module>\n    server = tornado.tcpserver.TCPServer(None, ssl_options=None, max_buffer_size=None)\nTypeError: __init__() got multiple values for argument \'ssl_options\'\n')])
  File "TCPServerNN.py", line 3, in <module>
Error inducing API short name: TCPServer, line no: 3
Error inducing API fqn: tornado.tcpserver.TCPServer, line no: 3
*** TypeError: PLACEHOLDER
Repair Action: arg_name
Key Params: ['IMPLICIT_0', 'ssl_options', 'max_buffer_size']
Prev Params: None
Prev Repair Action: None
Line No: 3
Prev Line No: 0
GitHub Candidates:
[]
KP:['IMPLICIT_0', 'ssl_options', 'max_buffer_size']
EXPP: ['IMPLICIT_0', 'ssl_options', 'max_buffer_size']
APIDoc Candidates:
[]
All Solution Candidates:
[]
[INFO] This case cannot be fixed!


real	0m0.426s
user	0m0.384s
sys	0m0.043s
